Settings.cpp

```diff
@@ -406,6 +411,8 @@ SoapySDR::RangeList SoapyRTLSDR::getFrequencyRange(
             results.push_back(SoapySDR::Range(22000000, 1100000000));
         } else if (tunerType == RTLSDR_TUNER_FC0013) {
             results.push_back(SoapySDR::Range(22000000, 948600000));
+        } else if (tunerType == RTLSDR_TUNER_R828D && strcmp(manufact, "RTLSDRBlog") == 0 && strcmp(product, "Blog V4") == 0) {
+            results.push_back(SoapySDR::Range(0, 1764000000));
```

Comment:
How about adding `// The RTL-SDR Blog V4 can tune down to 0 MHz (in reality ~300 kHz) because of the built in upconverter.`
It is the odd one out in this list and will surprise future readers otherwise. Otherwise good to merge anyway.

Reply:
Added comments